### PR TITLE
[AWS 4.12] AWS Local Zone CI profile

### PR DESCRIPTION
**Pull request by yunjiang29**
[AWS 4.12] AWS Local Zone CI profile

### CODE REVIEW

**Comment by yunjiang29** [comment]
All installs get passed.

The `cluster monitoring related upgrade check` [failed](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/34211/rehearse-34211-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-localzone-p2-f7/1595003269241901056/artifacts/aws-ipi-localzone-p2-f7/cucushift-upgrade-check/artifacts/upgrade-check/junit-report/TEST-features-upgrade-monitoring-monitoring-upgrade.xml),  which is a known issue https://issues.redhat.com/browse/OCPBUGS-3910.

@liangxia @jiajliu @jianlinliu @shellyyang1989 PTAL.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -389,6 +389,30 @@ tests:
...
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-localzone
+- as: aws-ipi-localzone-p2-f7-destructive
+  cron: 0 22 */7 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:405]: Do we really want to run destructive tests with f7 ?

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: fixed.

**Comment by shellyyang1989** [comment]
Is the Local Zone supported from 4.12? Will it be added to 4.12-4.13 upgrade test?

**Comment by yunjiang29** [comment]
> Is the Local Zone supported from 4.12?
Yes.

> Will it be added to 4.12-4.13 upgrade test?
It will be added along with 4.13 nightly e2e.

**Comment by shellyyang1989** [comment]
lgtm on the upgrade change

**Pull request merged**